hack/collector-default

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+function collect_sysinfo(){
+    cd ${BUNDLE_DIR}
+    mkdir -p hostinfos
+    cd hostinfos
+    
+    echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+    echo "os-release : " `cat ${HOST_PATH}/etc/os-release` >> hostinfo
+    echo "system-info : " `uname -a` >> hostinfo
+
+    # Boot Configuration
+    cat ${HOST_PATH}/boot/config-$(uname -r) > kernel_config
+}
+
+function collect_dmesg(){
+    dmesg -HTx &> dmesg.log
+}
+
+function collect_syslog(){
+    tail -c 10m ${HOST_PATH}/var/log/syslog > syslog.log
+}
+
+function collect_kubelet_log(){
+    # On Linux nodes that use systemd, the kubelet and container runtime write to journald by default. 
+    # ref. https://kubernetes.io/docs/concepts/cluster-administration/logging/#log-location-node
+    chroot ${HOST_PATH} /usr/bin/journalctl -r -u kubelet > kubelet.log
+    
+    # RKE2: Logs for the kubelet are save in rke2 agent log folder
+    RKE2_KUBELET_LOG_PATH=${HOST_PATH}/var/lib/rancher/rke2/agent/logs/kubelet.log
```

Comment:
nit: make `RKE2_KUBELET_LOG_PATH` as a global variable

Reply:
I have moved the variable outside the function as a global variable and done testing it works fine.